### PR TITLE
Added property god and staminaWhenUnwounded

### PR DESCRIPTION
**Pull request by Lion-alt**
### What?
Added property god and staminaWhenUnwounded
### Why?
Needed in character creation.
### How?
Edited PlayerDto.
### Testing?
Not needed.

### CODE REVIEW

**Review by Nilstrieb — APPROVED** [review]
cc @Noeell, you will need to adapt your frontend to this

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/service/PlayerSession.java [hunk 1]
```diff
@@ -88,6 +88,8 @@ public record PlayerDto(String name,
                             RankEnum rank,
                             ProfessionEnum profession,
                             int stamina,
+                            String god,
+                            int staminaWhenUnwounded,
```
Comment (keykey7) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/service/PlayerSession.java:92]: we'll need a rule such that stamina can never go above this value... but maybe in another PR were we tackle all game rules as a whole: it's getting scattered.

**Review by keykey7 — APPROVED** [review]

**Review by tamer545 — APPROVED** [review]
Looks good! Wow

**Review by kaibria — APPROVED** [review]
Wow! Fast and clean implementation

**Pull request merged**